model/service_logs/v1/cluster_log_type.model

```diff
@@ -63,4 +63,7 @@ class LogEntry {
 
 	// The related subscription id of the cluster.
 	SubscriptionID String
+
+	// Type of the service log entry.
+	LogType String
```

Comment:
based on the changes here - https://gitlab.cee.redhat.com/service/ocm-service-log/-/merge_requests/315/diffs#319143a9dee8d54c363706215f6d556d06602a1f

LogType expects an enum of the following. 

```
          - clustercreate-high-level
          - clustercreate-details
          - clusterremove-high-level
          - clusterremove-details
```

We should add this to the model too. Similar to how we do billing model https://github.com/openshift-online/ocm-api-model/blob/main/model/clusters_mgmt/v1/billing_model_type.model#L18

Reply:
@ciaranRoche  I went ahead created the enum model for log types.
Please advise if this is the correct approach.

FYI
It is still pending for a confirmation on additional types / values for the enum.

Reply:
You need to change the type from String to LogType so that it accepts the enum.
```suggestion
	LogType LogType
```

Reply:
Thank you!
I went ahead and fixed the type associated with Log Type, as suggested.